pages/02_auth_and_sso.md

```diff
@@ -8,27 +8,66 @@ In most embedding scenarios, you will want to enable single sign-on so that the
+1. POST Request: When the user navigates to a page in your web application that contains Tableau content, the web application will make a server-side POST request to Tableau Server passing in the users's Tableau Server username, the site the content exists on, and, optionally, the client's IP address in the form data. If the IP address making the request is trusted, and the user exists in Tableau Server, Tableau Server will return a ticket. [[Details]](https://onlinehelp.tableau.com/current/server/en-us/trusted_auth_webrequ.htm)
 1. Client loads the view with the ticket: Your web application now instructs the client to load the url of the desired resource, with the ticket inserted. If the ticket is valid, Tableau Server will start a session for the user and the user will see the visualization. Of course, the user does not see the HTTP requests going on behind the scenes, but simply loads a page in your application and sees embedded Tableau content without having to signin. [[Details]](https://onlinehelp.tableau.com/current/server/en-us/trusted_auth_webURL.htm)
 
 Additional considerations:
 
 * A common desire is to use a single 'service' account to authenticate the users. This is not a recommended approach, because it does not allow you to apply [data security]({{ site.baseurl }}/pages/04_multitenancy_and_rls) or to track usage on a per-user basis.
-* The trusted ticket is redeemable only once and the Tableau Server session is only valid for the visualization that was originally loaded. Therefore, your web application must request an additional ticket if refreshes the web page or navigates to a different page that contains embedded content.
-* By default, tickets can be redeemed only for visualizations, and not for other content pages in Tableau Server. To enable the user to see those, you must configure [unrestricted tickets](http://kb.tableau.com/articles/issue/login-prompt-when-embedding-server). See also: the [embedding non-view content]({{ site.baseurl }}/pages/06_embedding_non_view_content) page in this playbook.
-* If your web application has dynamic ip addresses, such that it is not feasible to trust a specific set of static ip addresses, you should create a small 'ticket requester' application that only allows requests from your web application, requests tickets from Server, and then returns them to your web application. You can then deploy this 'ticket requester' application to a static ip address.
+* The trusted ticket is redeemable only once and is valid for the current Tableau Server session. The session allows the user to access any of the views that the user would have if they logged onto the server. For more information, see [Trusted Authentication](https://help.tableau.com/current/server/en-us/trusted_auth.htm).
```

Comment:
Change this:
>* The trusted ticket is redeemable only once and is valid for the current Tableau Server session. The session allows the user to access any of the views that the user would have if they logged onto the server. For more information, see [Trusted Authentication](https://help.tableau.com/current/server/en-us/trusted_auth.htm).

to this:

* The trusted ticket is redeemable only once within three minutes of being issued and establishes a Tableau Server session for the user. The session allows the user to access any of the views that they have access to via Tableau Server User & Content permissions. For more information, see [Trusted Authentication](https://help.tableau.com/current/server/en-us/trusted_auth.htm).

Reply:
done